index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <a href="#" class="logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul Glad to see you!</h1>
+            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <h2 class="connect-info">LET’S CONNECT</h2>
```

Comment:
- [Optional] This should be a link `<a>` tag as it will be used to perform, an on-page navigation to the contact form in your project, which will be implemented in the future tasks on this project.